frontend/packages/console-shared/src/utils/multiselectdropdown.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import { Select, SelectOption, SelectVariant } from '@patternfly/react-core';
+
+export const MultiSelectDropdown: React.FC<MultiSelectDropdownProps> = ({
+  onChange,
```

Comment:
```suggestion
 onSelect
```
Let's keep the names consistent,